widget/label.go

```diff
@@ -36,9 +38,10 @@ func NewLabelWithData(data binding.String) *Label {
 // NewLabelWithStyle creates a new label widget with the set text content
 func NewLabelWithStyle(text string, alignment fyne.TextAlign, style fyne.TextStyle) *Label {
 	l := &Label{
-		Text:      text,
-		Alignment: alignment,
-		TextStyle: style,
+		Text:       text,
+		Alignment:  alignment,
+		TextStyle:  style,
+		Importance: MediumImportance,
```

Comment:
This should not be needed. `MediumImportance` is equal to zero and thus is the default.